shared/wallets/banner/index.tsx

```diff
@@ -41,6 +43,26 @@ const Banner = (props: Props) => (
           Please review.
         </Text>
       )}
+      {alert(JSON.stringify(props))}
+      {(props.offerAdvancedSendForm === AdvancedBanner.receiverBanner ||
+        props.offerAdvancedSendForm === AdvancedBanner.senderBanner) &&
+        props.onAction && (
+          <>
+            {props.offerAdvancedSendForm === AdvancedBanner.receiverBanner &&
+              'This user accepts more assets than XLM. '}
+            {props.offerAdvancedSendForm === AdvancedBanner.senderBanner &&
+              'You can send more assets than XLM. '}
+            <Text
+              type="BodySmallSemiboldPrimaryLink"
+              center={true}
+              style={styles.secondText}
+              negative={true}
+              onClick={props.onAction}
+            >
+              Send other assets?
```

Comment:
I'm mildly opposed to this question mark. It's already clear this is a link you are not required to tap. And there's no question that it will work when tapped.

Reply:
Sure. Do we want to put a period after it or just delete the question mark?